src/tests/ui/planner/workitem-list.ts

```diff
@@ -1,5 +1,5 @@
 import { BaseElement } from './../base.element';
-import { ElementFinder, by } from 'protractor';
+import { ElementFinder, by, browser } from 'protractor';
```

Comment:
Unnecessary import

Reply:
@jarifibrahim It is required https://github.com/Raunak1203/fabric8-planner/blob/30bcae1770e784035c3be0a14b934318d8fc3060/src/tests/ui/planner/workitem-list.ts#L74

Reply:
okay